test/libsolidity/semanticTests/userDefinedValueType/calldata_to_storage.sol

```diff
@@ -25,19 +25,19 @@ contract C {
 // ----
 // s() -> 0, 0, 0x00, 0
 // f((uint8,uint16,bytes2,uint8)): 1, 0xff, "ab", 15 ->
-// gas irOptimized: 110778
+// gas irOptimized: 44860
 // gas legacy: 112851
-// gas legacyOptimized: 110766
+// gas legacyOptimized: 44923
```

Comment:
This sounds like the gas costs were computed without the storage slot change?? How did it succeed in the PR?

Reply:
Yeah, this looks too suspicious. The cost is almost half.

Reply:
Ah no it's fine - the test was added to develop before we fixed anything.

Reply:
If that is the case, then I think the gas for legacy should also change.